tests/tests.py

```diff
@@ -1,11 +1,28 @@
 import argparse
+import os
 import subprocess
-from os.path import isfile
+from os.path import isdir, isfile, join
 from pathlib import Path
 
 _ROOT_DIR = Path(__file__).parents[1]
 
 
+def clear_getting_started_example_output_directory(output_directory):
```

Comment:
how about `tmpdir`? Then we wouldn't need this check right?

Reply:
I think it would be difficult to use `tmpdir`, since the getting started example is a "real-world" training run where the output file paths are explicitly specified in the config file.

Reply:
true we cannot do that at the moment. In the future, we could define a component, which provides you with a tmp path.